src/mjolnir/graphtilebuilder.cc

```diff
@@ -1295,10 +1295,12 @@ void GraphTileBuilder::AddLandmark(const GraphId& edge_id, const Landmark& landm
 
   if (eib == edgeinfo_offset_map_.end()) {
     throw std::runtime_error("Couldn't find edge info for the given edge: " +
-                             std::to_string(static_cast<int>(edge_id.id())));
+                             std::to_string(edge_id));
   }
 
+  // get the value and prepend the tag to it
   std::string tagged_value = landmark.to_str();
+  tagged_value.insert(tagged_value.begin(), static_cast<char>(baldr::TaggedValue::kLandmark));
```

Comment:
both the encoding of a landmark to binary and decoding of the landmark to binary do not include the preceeding "tag" character now. it was quite confusing when testing that one had it and the other didn't